### ISSUE TITLE
can't build with Sphinx v1.6.4

### ISSUE THREAD

**Issue by bjuergens** [openmicroscopy/ome-documentation#1791]
steps to reproduce on ubuntu 17.04:
```
git clone git@github.com:openmicroscopy/ome-documentation.git
sudo apt install ant
sudo pip install --upgrade Sphinx
cd omero
make html
```

result:

```
ant html     
Buildfile: /home/bjoern/dev/project/ome-documentation/omero/build.xml

sphinx-deps:

init:

html:
     [exec] Running Sphinx v1.6.4
     [exec] 
     [exec] Configuration error:
     [exec] There is a syntax error in your configuration file: Missing parentheses in call to 'print' (conf.py, line 359)
     [exec] Did you change the syntax from 2.x to 3.x?

BUILD FAILED
/home/bjoern/dev/project/ome-documentation/common/rules.xml:75: The following error occurred while executing this line:
/home/bjoern/dev/project/ome-documentation/common/rules.xml:42: exec returned: 1

Total time: 0 seconds
../Makefile:23: die Regel für Ziel „html“ scheiterte
make: *** [html] Fehler 1
```

is there a specific version of Sphinx required to build this?

**Comment by bjuergens** [comment]
workaround for me: download _OMERO.doc-5.4.0.zip_ from https://downloads.openmicroscopy.org/omero/5.4.0/artifacts/

(I just wanted a offline version)

**Comment by sbesson** [comment]
@wotanii: we worked on fixing `ome-documentation` to work with the Sphinx 1.6 and the [Travis build](https://github.com/openmicroscopy/ome-documentation/blob/develop/.travis.yml) should test the documentation builds with the latest version.

Given the stack trace, your error seems rather related to the usage of Python 3 while we are currently building on Python 2.7. We have not reviewed the changes to support building with Python 3 and this is currently not high-priority but it might be small if anyone wants to contribute with a Python 2 backwards-compatible Pull Request.

For retrieving an existing version, as you rightfully mentioned, the documentation for every OMERO version is available :

- either as a zip artifact under https://downloads.openmicroscopy.org/omero/"version"/artifacts/
- or as a web documentation under https://docs.openmicroscopy.org/omero/"version"/

Closing this issue as this is unrelated to the Sphinx version.
